styling/src/jsMain/kotlin/dev/fritz2/styling/theme/default.kt

```diff
@@ -2068,4 +2068,116 @@ open class DefaultTheme : Theme {
                 }
             }
     }
+
+    override val slider = object : SliderStyles {
+
+        val self = this
+
+        override val sizes = object : FormSizes {
+            override val large: Style<BasicParams> = {
+                css("--sl-size: 1.5rem")
```

Comment:
I would insert a second css var for the spacing that is used. So you have for every size a configurable spacing.
Further in styling you can use the css-var instead of the value and  it will be forgotten when you wanna change the spacing.

Reply:
solved in https://github.com/jwstegemann/fritz2/pull/387/commits/c03de92707ffae917d5de095fda227938a177780